doc/tech/usecases.md

```diff
@@ -2,22 +2,44 @@
 # CNTT Use Cases
 
 ## Table of Contents
- * [5.1 Core](#5.1)
- * [5.2 Edge](#5.2)
+ * [5.1 Edge](#5.1)
+   * [5.1.1 Executive Summary](#5.1.1)
+   * [5.1.2 Objective](#5.1.2)
+   * [5.1.3 Approach & Scope](#5.1.3)
+   * [5.1.4 Principle](#5.1.4)
+   * [5.1.5 Use cases](#5.1.5)
+   * [5.1.6 Terminologies](#5.1.6)
+   * [5.1.7 SDO interlocking](#5.1.7)
+
    
 <a name="5.1"></a>
-## 5.1 Core
+## 5.1 Edge
 
+<a name="5.1.1"></a>
+### 5.1.1 Executive Summary
 
-<a name="5.1"></a>
-## 5.1 Edge
+Edge computing considered one of disruptive  technology should be considered by any Digital Service provider (DSP) as it is tightly linked with a lot of use cases that can monetize the 5G adoption , therefor a lot of CSP , Public  cloud , content provider and application developer put focus on Edge 
```

Comment:
Have we defined Edge computing? If not then somewhere we need a definition. 

The existing sentence needs some rewording and deletion of spaces before commas. Maybe as in the following:

"Edge computing, a disruptive  technology, should be considered for adoption by Digital Service Providers (DSP) as it is tightly linked with a number of use cases that monetize 5G.  Thus, operators, cloud service providers, content providers and application developers are increasingly focussed on Edge computing."

Reply:
done